src/app/newstudents/page.tsx

```diff
@@ -0,0 +1,55 @@
+import { SectionTitle } from "@components/SectionTitle";
+import { InfoCard } from "@components/InfoCard";
+import { STORIES, TODAY_STATISTICS } from "@data";
+import { Sponsors } from "@components/Sponsors";
+import { TitleSubtitle } from "@components/TitleSubtitle";
+
```

Comment:
Add some margin to this, I'm not a fan of text stretching across the screen so wide, it makes it harder to read, especially in thick paragraphs. This is a site wide issue tbh